.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,28 @@
+name: Feature request
+description: Ask for a feature to be implemented and included in the next release
+labels: feature request
+title: "[REQUEST] "
```

Comment:
These prefixes are a bit redundant with the labels. Can we drop one of the
two? On the GitHub Web UI the labels are clearly visible.  GitHub email
notifications don't contain the labels though.